example/templates/day5.ts

```diff
@@ -0,0 +1,194 @@
+      ]
+    },
+    {
+      'type': 'Control',
+      'scope': {
+        '$ref': '#/properties/description'
+      },
+      'options': {
+          'multi': true
+      }
+    },
+    {
+      'type': 'HorizontalLayout',
+      'elements': [
+        {
+          'type': 'Control',
+          'scope': {
+            '$ref': '#/properties/recurrence'
+          }
+        },
+        {
+          'type': 'Control',
+          'scope': {
+            '$ref': '#/properties/recurrence_interval'
+          },
+          'rule': {
+              'effect': 'HIDE',
+              'condition': {
+                  'scope': {
+                      '$ref': '#/properties/recurrence'
+                  },
+                  'expectedValue': 'Never'
+              }
+          }
+        }
+      ]
+    }
+  ]
+};
+const uischema_category = {
+  'type': 'Categorization',
+  'elements': [
+    {
+      'type': 'Category',
+      'label': 'Main',
+      'elements': [
+        {
+          'type': 'Control',
+          'label': false,
+          'scope': {
+            '$ref': '#/properties/done'
+          }
+        },
+        {
+          'type': 'Control',
+          'scope': {
+            '$ref': '#/properties/name'
+          }
+        },
+        {
+          'type': 'Control',
+          'scope': {
+            '$ref': '#/properties/description'
+          },
+          'options': {
+              'multi': true
+          }
+        }
+      ]
+    },
+    {
+      'type': 'Category',
+      'label': 'Additional',
+      'elements': [
+        {
+          'type': 'HorizontalLayout',
+          'elements': [
+            {
+              'type': 'Control',
+              'scope': {
+                '$ref': '#/properties/due_date'
+              }
+            },
+            {
+              'type': 'Control',
+              'scope': {
+                '$ref': '#/properties/rating'
+              }
+            }
+          ]
+        },
+        {
+          'type': 'HorizontalLayout',
+          'elements': [
+            {
+              'type': 'Control',
+              'scope': {
+                '$ref': '#/properties/recurrence'
+              }
+            },
+            {
+              'type': 'Control',
+              'scope': {
+                '$ref': '#/properties/recurrence_interval'
+              },
+              'rule': {
+                  'effect': 'HIDE',
+                  'condition': {
+                      'scope': {
+                          '$ref': '#/properties/recurrence'
+                      },
+                      'expectedValue': 'Never'
+                  }
+              }
+            }
+          ]
+        }
+      ]
+    }
+  ]
+};
+const data = {
+  'name': 'Send email to Adriana',
+  'description': 'Confirm if you have passed the subject\nHerby ...',
+  'done': true,
+  'recurrence': 'Daily'
+};
+registerExamples([
+  {name: 'day5', label: 'Day 5', data: data, schema: schema, uiSchema: uischema},
```

Comment:
no, there is no group in day5 but day5 was about layouts and group is one of the layouts we support